components/messaging/ConversationsList.tsx

```diff
@@ -7,17 +7,24 @@ import { ConversationCardType } from '@/utils/messaging/messagingTypes';
 import DeleteConvoModal from '../DeleteConvoModal';
 
 const ConversationsList: React.FC = () => {
-  const { allConversations, setAllConversations, setCurrentConversation } =
-    useContext(useConversation);
+  const {
+    allConversations,
+    setAllConversations,
+    setCurrentConversation,
+    setShowConversationList,
+  } = useContext(useConversation);
   const supabase = createSupabaseClient;
 
   const updateOpenConvo = async (givenId: number) => {
+    console.log('update conversation');
     setCurrentConversation &&
       setCurrentConversation(
         allConversations?.filter(
           (conversations) => conversations.conversation_id === givenId
         )[0]
       );
+
+    setShowConversationList(false);
```

Comment:
should we evaluate first if shoeConversationList is already false? The useEffect would kick even if we are on desktop and then be setting a state needlessly, no?